tests/Unit/UpcomingOpeningTimeTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Rapidez\SmileStoreLocator\Tests\Unit;
+
+use Carbon\Carbon;
+use Illuminate\Support\Str;
+use Rapidez\SmileStoreLocator\Models\Retailer;
+use Rapidez\SmileStoreLocator\Models\Times;
+use Rapidez\SmileStoreLocator\Tests\TestCase;
+
+class UpcomingOpeningTimeTest extends TestCase
+{
+    public function test()
+    {
+        config(['frontend.day_names' => collect(Carbon::getDays())->map(function ($day, $index) {
+            return ucfirst(Carbon::create(Carbon::getDays()[$index])->dayName);
+        })->toArray()]);
+
+        $tests = [
+            0 => Carbon::now()->addHour()->format('H:i'),
+            1 => strtolower(config('frontend.day_names')[today()->addDay()->dayOfWeek]),
+            2 => Carbon::now()->addHour()->format('H:i'),
+            3 => strtolower(config('frontend.day_names')[today()->addDay()->dayOfWeek])
+        ];
+
+        $retailers = $this->generateTestRetailers();
+
+        foreach ($tests as $key => $result) {
+            $this->assertEquals($retailers[$key]->upcoming_opening_time, $result);
```

Comment:
This isn't very readable at the moment. The results are defined in the array here and the configuration bellow. Can't we have this in one array or splitted in multiple test methods so we can see what results in what?